webanno-ui-curation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/curation/util/MergeCas.java

```diff
@@ -491,8 +491,7 @@ public static void setFeatureValue(FeatureStructure aFS, Feature aFeature, Objec
             aFS.setShortValue(aFeature, Short.valueOf(aValue.toString()));
             break;
         default:
-            return;
-        // return aFS.getFeatureValue(aFeature);
+            // return aFS.getFeatureValue(aFeature);
```

Comment:
I find having the explicit return here is better in terms of readability than treating default as a fall-through case.